controller/subscriber.go

```diff
@@ -30,6 +30,6 @@ func (s Subscription) Reconcile(ctx context.Context, resourceEvents []ResourceEv
 		})
 	})
 	if len(matchingEvents) > 0 && s.ReconcileFunc != nil {
-		s.ReconcileFunc(ctx, matchingEvents, topology)
+		s.ReconcileFunc(ctx, matchingEvents, topology, nil)
```

Comment:
Why `nil`? I would expect `err` passed along here.

```suggestion
		s.ReconcileFunc(ctx, matchingEvents, topology, err)
```